kroxylicious-integration-tests/src/test/java/io/kroxylicious/proxy/MetricsIT.java

```diff
@@ -259,6 +301,198 @@ void shouldIncrementConnectionMetrics(KafkaCluster cluster, Topic topic) throws
             assertMetricsWithValue(updatedMetricsList, "kroxylicious_downstream_connections_total", null);
             assertMetricsWithValue(updatedMetricsList, "kroxylicious_upstream_connections_total", null);
         }
+        catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Test
+    void shouldIncrementUpstreamConnectionErrorMetrics(KafkaCluster cluster, Topic topic) {
```

Comment:
test name is `shouldIncrementUpstreamConnectionErrorMetrics` but the test itself is actually testing the case where there are no errors, isn't it?